### PR TITLE
Tickers tests

### PR DESCRIPTION
**Pull request by Darcy-Linde**

### CODE REVIEW

File: polygon/rest/models/tickers.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ class Ticker:
     cik: Optional[str] = None
     composite_figi: Optional[str] = None
     currency_name: Optional[str] = None
+    currency_symbol: Optional[str] = None
+    base_currency_symbol: Optional[str] = None
+    base_currency_name: Optional[str] = None
```
Comment (clickingbuttons) [inline polygon/rest/models/tickers.py:36]: Can you make a note that these are are undocumented?

File: polygon/rest/models/tickers.py [hunk 2]
```diff
@@ -57,6 +60,7 @@ class TickerDetails:
     currency_name: Optional[str] = None
     delisted_utc: Optional[str] = None
     description: Optional[str] = None
+    ticker_root: Optional[str] = None
```
Comment (clickingbuttons) [inline polygon/rest/models/tickers.py:63]: Same here.

Comment (Darcy-Linde) [reply polygon/rest/models/tickers.py:63]: Yep did that too

**Review by clickingbuttons — APPROVED** [review]
Ty!

**Pull request merged**